redash/query_runner/databricks.py

```diff
@@ -26,67 +59,98 @@ def configuration_schema(cls):
             "type": "object",
             "properties": {
                 "host": {"type": "string"},
-                "database": {"type": "string"},
                 "http_path": {"type": "string", "title": "HTTP Path"},
+                # We're using `http_password` here for legacy reasons
                 "http_password": {"type": "string", "title": "Access Token"},
+                "schemas": {"type": "string", "title": "Schemas to Load Metadata For"},
             },
-            "order": ["host", "http_path", "http_password", "database"],
+            "order": ["host", "http_path", "http_password"],
             "secret": ["http_password"],
-            "required": ["host", "database", "http_path", "http_password"],
+            "required": ["host", "http_path", "http_password"],
         }
 
-    def _get_connection(self):
-        host = self.configuration["host"]
+    def _get_cursor(self):
+        connection_string = _build_odbc_connection_string(
```

Comment:
Worth documenting where this came from in the [DB Documentation](https://docs.databricks.com/integrations/bi/jdbc-odbc-bi.html#odbc-data-source-name-dsn-configuration-for-the-simba-odbc-driver)?

Reply:
It's not from there :) It's from the tableau connector implementation.